src/Symfony/Component/Form/Tests/Extension/Core/Type/CheckboxTypeTest.php

```diff
@@ -80,15 +80,15 @@ public function testSubmitWithValueChecked()
         $this->assertEquals('foobar', $form->getViewData());
     }
 
-    public function testSubmitWithRandomValueChecked()
+    public function testSubmitWithRandomValueUnchecked()
```

Comment:
Can you change "Random" ot "Unknown"? Technically, "random" is not correct here.